### ISSUE TITLE
News Posts can't be quickly added

### ISSUE THREAD

**Issue by Naviary2** [Infinite-Chess/infinitechess.org#250]
With the current language foundation, news posts cannot be quickly added to the production website, because they require you to modify the html, and atleast the English TOML file, which may or may not already be modified in the developmental version.

## Proposal: ##

Have each news post be contained in a single file, written in markup, and all contained in a blog/news folder inside `translation/`, one for each language. Then, the website automatically adds them to the news page. This would eliminate the need to write more code whenever we add a post, just needing to drag and drop a new post file.

**Comment by Heinrich-XIAO** [comment]
So, there are multiple post files? If so, how would they be translated quickly?

**Comment by Naviary2** [comment]
> So, there are multiple post files? If so, how would they be translated quickly?

Probably, a folder for each language's news posts.

**Comment by Heinrich-XIAO** [comment]
But it would take a non zero amount of time to translate

**Comment by Heinrich-XIAO** [comment]
Maybe they could be AI translated and then updated when there is a translator available

**Comment by Naviary2** [comment]
> Maybe they could be AI translated and then updated when there is a translator available

It would be nice if we found a way to automate the AI translations. Like, we would just run a command to have the AI translate any non-translated posts.

If a news post hasn't been translated yet, then it will just be in english, even on other languages.

**Comment by Idonotus** [comment]
Might want to look into google translate api

**Comment by Heinrich-XIAO** [comment]
> Have each news post be contained in a single file, written in markup
Readme markup or hypertext markup language markup?

**Comment by Naviary3** [comment]
> Readme markup or hypertext markup language markup? HTML would be easier to write code for displaying, but readme is might be easier to write.

Yeah I would go with markdown (which I believe is the same thing used in readme's? I also confused the name with markup, I mean markdown). It's also super easy to add code blocks by just adding triple backticks ```

**Comment by Heinrich-XIAO** [comment]
Ready for review #257
